### PR TITLE
chore: docs moving around

### PR DESCRIPTION
**Pull request by daKmoR**

### CODE REVIEW

File: docs/blog/introducing-modern-web/index.md [hunk 1]
```diff
@@ -124,10 +124,12 @@ While we have now spread out over multiple repos like Open WC, Modern Web, and R
 
 ## Thanks for reading
 
-We are incredibly proud of our first Modern Web Tool, and we hope you find it useful as well. If you find an issue or if you are stuck [please let us know](https://github.com/modernweb-dev/web/issues/new).
+We are incredibly proud of what we archived so far and the direction we are heading and we invite you to join us on our path forward.
```
Comment (LarsDenBakker) [inline docs/blog/introducing-modern-web/index.md:127]:
```suggestion
We are incredibly proud of what we have achived so far and the direction we are heading and we invite you to join us on our path forward.
```

File: docs/blog/index.md [hunk 2]
```diff
@@ -14,6 +14,6 @@ pagination:
 
 Discover articles from the core team and contributors about Modern Web, tips, and tricks included!
 
-You want to write a blog post or you want to write about a specific topic? [Suggest it!](https://github.com/modernweb-dev/web/issues/new?title=[blog%20post]%20Write%20about%20...&labels=blog%20post&body=I%20would%20like%20to%20write%20about...)
+You want to write a blog post or you want use to write about a specific topic? [Suggest it!](https://github.com/modernweb-dev/web/issues/new?title=[blog%20post]%20Write%20about%20...&labels=blog%20post&body=I%20would%20like%20to%20write%20about...)
```
Comment (LarsDenBakker) [inline docs/blog/index.md:17]:
```suggestion
Do you want to write a blog post, or want us to write about a specific topic? [Suggest it!](https://github.com/modernweb-dev/web/issues/new?title=[blog%20post]%20Write%20about%20...&labels=blog%20post&body=I%20would%20like%20to%20write%20about...)
```

File: docs/blog/introducing-modern-web/index.md [hunk 3]
```diff
@@ -38,7 +38,7 @@ We have been working on a lot of different projects in the last couple of years.
 
 ## Guides
 
-On our all-new [website](https://modern-web.dev), we've included a ["Guide"](../../guides/web-development/getting-started.md). Here we feature step by step guides to work with our tools, and we document common issues developers run into when doing buildless development.
+On our all-new [website](https://modern-web.dev), we have a dedicated [Guides](../../guides/index.md) section. It is dedicated in helping you to get confident in our modern web way of working. It feature step by step guides to work with our tools, and we documented common issues developers run into when doing buildless development.
```
Comment (LarsDenBakker) [inline docs/blog/introducing-modern-web/index.md:41]:
```suggestion
On our all-new [website](https://modern-web.dev), we have a dedicated [Guides](../../guides/index.md) section. It is meant to help you to get confident building for the modern web. It features step by step guides to work with our tools, and we document common issues developers run into when doing buildless development.
```

File: docs/blog/introducing-modern-web/index.md [hunk 4]
```diff
@@ -106,15 +106,15 @@ Not only do we ship [rollup-plugin-workbox](https://www.npmjs.com/search?q=rollu
 
 In the past years, we've used different frameworks and tools to build our websites. Recently we've become big fans of [11ty](https://www.11ty.dev/). It's simple to use, works with markdown, and generates _just_ plain HTML. There is no runtime javascript involved with displaying page content, making it super fast.
 
-To add interactivity to our page we started using web components, and applied rollup with our [HTLM](https://www.npmjs.com/package/@open-wc/rollup-plugin-html) and [workbox](https://www.npmjs.com/search?q=rollup-plugin-workbox) plugins to the HTML pages generated by 11ty.
+To add interactivity to our page we started using web components, and applied rollup with our [HTML](https://www.npmjs.com/package/@open-wc/rollup-plugin-html) and [workbox](https://www.npmjs.com/search?q=rollup-plugin-workbox) plugin.
```
Comment (LarsDenBakker) [inline docs/blog/introducing-modern-web/index.md:109]:
```suggestion
To add interactivity to our page we started using web components, and applied rollup with our [HTML](https://www.npmjs.com/package/@open-wc/rollup-plugin-html) and [workbox](https://www.npmjs.com/search?q=rollup-plugin-workbox) plugins.
```

Comment (daKmoR) [reply docs/blog/introducing-modern-web/index.md:109]: `and applied rollup with our html and workbox plugins.`  sounds strange to me - but no clue 😅

Comment (LarsDenBakker) [reply docs/blog/introducing-modern-web/index.md:109]: Well it's plural :) The original sentence was `and applied rollup with our HTML and workbox plugins to the HTML pages generated by 11ty`. Not great either, so maybe we should rephrase it entirely 🤔

Comment (LarsDenBakker) [reply docs/blog/introducing-modern-web/index.md:109]: `and added rollup using our HTML and workbox plugins`?

Comment (daKmoR) [reply docs/blog/introducing-modern-web/index.md]: change it to 

To add interactivity to our page we started using web components and applied rollup with our plugins for [HTML](https://www.npmjs.com/package/@open-wc/rollup-plugin-html) and [workbox](https://www.npmjs.com/search?q=rollup-plugin-workbox).

sounds good?

Comment (LarsDenBakker) [reply docs/blog/introducing-modern-web/index.md]: Yeah 👍

**Pull request merged**